transcom/mymove
# INT-B-20466 add a frontend (filterable) column to the SC's queue

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A979856)

## Summary
Add a frontend column to the SC's queue that displays move.counselingOffice, and can be filtered and sorted.

### How to test

1.  Login as a Service Counselor
2. Search for a customer using a customer name
3. Create a new move for the customer
4. Verify Counseling Office column shows respective counseling office name
5. Enter the name of the Counseling Office and verify only respective moves are shown 

### 508 Screenshots
[508Screenshots.pdf](https://github.com/user-attachments/files/16896412/508Screenshots.pdf)
